src/main/webapp/dataverse.xhtml

```diff
@@ -135,7 +136,7 @@
                                 <div class="form-col-container">
                                     <div class="input-group">
                                         <div class="input-group-addon"><span class="small">#{systemConfig.dataverseSiteUrl}/dataverse/</span></div>
-                                        <p:inputText id="identifier" styleClass="form-control" validator="#{DataversePage.validateAlias}" value="#{DataversePage.dataverse.alias}" required="true"/>
+                                        <p:inputText id="identifier" styleClass="form-control" validator="#{DataversePage.validateAlias}" value="#{DataversePage.dataverse.alias}" />
```

Comment:
Why remove required on this (and other) fields?

Reply:
The required is handled by the @NotBlank bean validation on the fields in Dataverse. All of them are annotated with NotBlank

Reply:
again, returned the required=true based on Mike's need of them for accessibility.